### ISSUE TITLE
[BUG]: Urgent! Can not export model to tflite

### ISSUE THREAD

**Issue by nhat108** [googlecreativelab/teachablemachine-community#212]
** Can not export model to tflite**
 When i export model to tflite, it downloaded as zip file. but it can't be unzip. the file size is only 294 bytes.
Note: I had train 9 labels, each class have 100 images. Please check it as soon as possible!
<img width="478" alt="Screen Shot 2021-05-26 at 10 27 33" src="https://user-images.githubusercontent.com/40523356/119598348-fbe66f00-be0c-11eb-8183-f13edd5dcd5b.png">

**Comment by tttt5955736** [comment]
i have same problem T^T ,i still can't understand. hoping have someone can reslove

**Comment by nhat108** [comment]
You can download tensortflow model-> extra zip file -> upload the file .h5 to codelab and convert to .tflite file as example below:
`import tensorflow as tf

converter = tf.compat.v1.lite.TFLiteConverter.from_keras_model_file('/content/keras_model.h5')

tflite_model = converter.convert()

with open('model.tflite', 'wb') as f:
  f.write(tflite_model)`

> i have same problem T^T ,i still can't understand. hoping have someone can reslove

**Comment by tttt5955736** [comment]
i done.
but how to download the file which convert done?

**Comment by tttt5955736** [comment]
i try
but it write "No training configuration found in the save file, so the model was *not* compiled. Compile it manually."
T^T why0.0

**Comment by HalfdanJ** [comment]
This should hopefully have been resolved now. Duplicate of #225
